specificatie/domain.yaml

```diff
@@ -203,6 +203,15 @@ components:
                         De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin dit beslag is vermeld
           items:
             type: "string"
+    BijbehorendAppartementsrecht:
+      type: "object"
+      description: |
+                    De appartementsrechten die bij deze KadastraalOnroerende Zaak (grondperceel) horen
+      properties:
+        kadastraalOnroerendeZaakIdentificatie:
```

Comment:
een appartementsrecht is een KOZ, dan kan dit volgens mij worden hernoemd naar identificatie. Of zie ik dat verkeerd?

Reply:
mee eens